backend/src/App.ts

```diff
@@ -0,0 +1,83 @@
+import BodyParser from 'body-parser';
```

Comment:
Try to add some environment variables. And create file consts.ts - which includes some example environment variables. Also, create a declarations.d.ts file - check frontend